### ISSUE TITLE
paddle支持batch normalztion 双向rnn吗？并且带有clip relu的功能？

### ISSUE THREAD

**Issue by guoying1030** [PaddlePaddle/Paddle#1340]
paddle支持batch normalztion 双向rnn吗？并且带有clip relu的功能？

**Comment by qingqing01** [comment]
不是特别理解 "batch normalization 双向rnn" 是如何操作的，您可以再详细描述下嘛？ 或者给个参考也可以。 "带有clip relu" 指的是什么呢？ 对ReLU激活后做clip吗？

**Comment by luotao1** [comment]
@guoying1030 请问您说的是“Batch Renormalization: Towards Reducing Minibatch Dependence in Batch-Normalized Models”么？

**Comment by QiJune** [comment]
长时间没有更新，暂时close；如有进一步更新，欢迎reopen
